frontend/src/components/home/Home.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button } from '@mui/material';
+import Logo from '../logo/Logo';
+
+function Home(): JSX.Element {
+  return (
+    <div className="flex justify-center items-center flex-col">
+      <div className="flex">
+        <h1 className="text-9xl font-extrabold px-4 py-2 text-[#007FEF]">Buy.</h1>
+        <h1 className="text-9xl font-extrabold px-4 py-2 text-[#7E27C7]">Sell.</h1>
+        <h1 className="text-9xl font-extrabold px-4 py-2 text-[#FF544B]">Trade.</h1>
+      </div>
+      <p className="text-lg mt-2 font-normal text-inter text-[#6F6F6F] mt-2">
```

Comment:
Add this grey color as a secondary color in the mui theme in `App.tsx` and use it here properly.